### ISSUE TITLE
panic: invalid escape sequence

### ISSUE THREAD

**Issue by Travis-z** [github/gh-ost#467]
I started a gh-ost process with this command:

```
gh-ost \
--conf='/etc/my.cnf' \
--port=3306 \
--host='xxx.xx.xxx.xx' \
--user='user_name' \
--password='xxxxx' \
--database='xxxx' \
--table='xxxxxx' \
--alter='MODIFY COLUMN filed_name text' \
--verbose \
--initially-drop-ghost-table \
--switch-to-rbr \
--execute 
panic: invalid escape sequence

goroutine 1 [running]:
github.com/github/gh-ost/vendor/gopkg.in/gcfg%2ev1.unquote(0xc420111540, 0x12, 0x5, 0xc420111540)
	go/src/github.com/github/gh-ost/vendor/gopkg.in/gcfg.v1/read.go:32 +0x28f
github.com/github/gh-ost/vendor/gopkg.in/gcfg%2ev1.readInto(0x6fa1a0, 0xc4200c4040, 0xc420017880, 0xc420082540, 0xc4200c9000, 0xb45, 0xe00, 0xc400000008, 0xc420016100)
	go/src/github.com/github/gh-ost/vendor/gopkg.in/gcfg.v1/read.go:131 +0xae2
github.com/github/gh-ost/vendor/gopkg.in/gcfg%2ev1.ReadFileInto(0x6fa1a0, 0xc4200c4040, 0x7fff479b794f, 0xb, 0x0, 0x0)
	go/src/github.com/github/gh-ost/vendor/gopkg.in/gcfg.v1/read.go:187 +0x1b7
github.com/github/gh-ost/go/base.(*MigrationContext).ReadConfigFile(0xc4200c4000, 0x0, 0x0)
	go/src/github.com/github/gh-ost/go/base/context.go:684 +0xcd
main.main()
	go/src/github.com/github/gh-ost/go/cmd/gh-ost/main.go:204 +0x1688
```
Thank you so much for the help and for this amazing tool!
PS： 
filed_name varchar(1024)
CentOS release 5.7 (Final)，
gh-ost  version 1.0.40

**Comment by shlomi-noach** [comment]
Seems like the problem is with your config file `/etc/my.cnf`. Can you please see https://github.com/github/gh-ost/blob/master/doc/command-line-flags.md#conf ?

Maybe your config file has some unrelated params with weird characters or unquoted text or something?

**Comment by arthurnn** [comment]
@shlomi-noach should we improve the error here, and show the invalid sequence? or at least say the line on my.cnf?

**Comment by shlomi-noach** [comment]
@arthurnn we can certainly let the user know the problem is with parsing the config file. I wouldn't want to further edit the `gcfg.v1` package itself.

**Comment by shlomi-noach** [comment]
Addressed by #494
